### DISCUSSION TITLE
ZepetoCharacterCreator로 캐릭터를 만들때, 커스터마이징은 안될까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#716] [5. Q&A]
현재 별도의 NPC를 만들고 싶은데, 해당 코드로 NPC를 만드는 것은 결국 ID가 있어야 하고, 또 해당 아이디의 캐릭터가 커스터미이징을  변경하면 의도와 맞지 않는 디자인이 될 수도 있더군요. 

해서 체형까지는 무리가 있더라도 자체적으로 만들거나 제페토내에서 구입한 오브젝트, 옷을 코드를 통해 변경해서 생성할 수 있으면 좋겠습니다.

혹은 해당 오브젝트들을 다운로드 한 뒤 플레이어에게 적용할 수 있는 코드를 만들어 주실수는 없으실까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 위 내용은 많은 분들이 요청 주신 입어보기/갈아입기 관련 API를 통해 이용할 수 있을 것 같습니다.
해당 기능은 제공 예정이나 적용 시기는 현재로서는 공유드리기 어려운 점 이해 부탁 드립니다.
감사합니다.
